application/controllers/ParticipantController.php

```diff
@@ -215,15 +215,21 @@ public function surveyAction() {
             }
             $result = $db->fetchRow($db->select()
                 ->from(array('spm' => 'shipment_participant_map'), array('spm.map_id', 'spm.cs_survey_response', "spm.shipment_test_report_date"))
-                ->join(array('s' => 'shipment'), 's.shipment_id=spm.shipment_id', array('s.shipment_code', 's.cs_survey'))
+                ->join(array('s' => 'shipment'), 's.shipment_id=spm.shipment_id', array('s.shipment_code', 's.cs_survey', 's.finalized_date'))
                 ->join(array('p' => 'participant'), 'p.participant_id=spm.participant_id', array('p.lab_name'))
                 ->where("spm.map_id = ?", $map_id));
 
             if (!$result) {
                 $this->_redirect("/participant/report");
             }
 
-            $this->view->expired = (new DateTime($result['shipment_test_report_date']))->modify('+2 weeks')->diff(new DateTime())->days > 14;
+            $configFile = APPLICATION_PATH . '/configs/config.local.ini';
+            if (!is_file($configFile)) {
+                $configFile = APPLICATION_PATH . '/configs/config.ini';
+            }
+            $config = new Zend_Config_Ini($configFile, APPLICATION_ENV);
```

Comment:
```suggestion
            $configFile = APPLICATION_PATH . DIRECTORY_SEPARATOR . "configs" . DIRECTORY_SEPARATOR . "config.local.ini";
            if (!is_file($configFile)) {
                $configFile = APPLICATION_PATH . DIRECTORY_SEPARATOR . "configs" . DIRECTORY_SEPARATOR . "config.ini"';
            }
            $config = new Zend_Config_Ini($configFile, APPLICATION_ENV);
```
This is just to keep the ability to build & test in Windows